build/__dotNetCoreRunTimeVersions.sh

```diff
@@ -30,7 +30,7 @@ NET_CORE_APP_70='7.0.9'
 NET_CORE_APP_70_SHA='09552e5ae6ac014dadf17545ff0a30ab32921075a31fb33e7be148c13078e30d097f592ffa1b8d306563aaa3f6302e40c5c0ba815c1473bbd5d72e3bef55d91e'
 ASPNET_CORE_APP_70='7.0.9'
 ASPNET_CORE_APP_70_SHA='aabf4fa5ca726dc52774e5d644800ef7477815b22a982b7a2752dec6569186aabca93d5386e195e7ead377144601a786ae6a5d76ff28435bdabfad495cfe554b'
-NET_CORE_APP_80='8.0.0-preview.7.23375.6'
-NET_CORE_APP_80_SHA='bfd8491550178b86a7a72fe06bdc82f0dd66771d5b60d7e4e1133cdde29f84bd57857d846722e027bd209db087123b2d12b2e23590d77991052269fa265814e1'
-ASPNET_CORE_APP_80='8.0.0-preview.7.23375.9'
-ASPNET_CORE_APP_80_SHA='b8c8a5cd579a8ee1e082363a73a05a745499365445f784e6ff87547f9acdbe8b7ba525140ef10555bc2802c13af131c2d568ec6af020bd0dd2fdf82d4c258442'
+NET_CORE_APP_80='8.0.0-rc.1.23419.4'
```

Comment:
Nit: looks like we need to rebase the PR to get the latest changes you pushed in parallel.

Reply:
i think this one should be ok. The .NET changes were merged into this branch, and I merge all the changes (automator change and .NET 8 RC change) into main.

Reply:
Harry and I synced this morning. I indeed missed that the previous PR wasn't targeting main.